config/initializers/shrine.rb

```diff
@@ -7,10 +7,33 @@
 Shrine.plugin :direct_upload
 Shrine.plugin :determine_mime_type
 
-Shrine.storages = {
-  cache: Shrine::Storage::FileSystem.new("tmp", prefix: "uploads/cache"), # temporary
```

Comment:
For heroku we need to use the tmp directory because you are not allow to write to /public or /temp